README.md

```diff
@@ -177,74 +177,73 @@ export const query = graphql`
 ## 🔌 Plugin Options
 
 ### `cloudName` (required)
+You'll find your Cloudinary account's `cloudName` in your [Cloudinary console](https://cloudinary.com/console/).
 
-The Cloud Name of your Cloudinary account. You'll find it in your [Cloudinary console](https://cloudinary.com/console/).
-
-**Type:** String  
-**Default:** n/a  
-**Note:** This should be stored and retrieved as an environment variable.
+**Type:** String
+**Default:** n/a
+**Note:** Store and retrieve your `cloudName` as an environment variable.
 
 ### `apiKey` (required)
 
 The API Key of your Cloudinary account. You'll find it in your [Cloudinary console](https://cloudinary.com/console/).
 
-**Type:** String  
-**Default:** n/a  
-**Note:** This should be stored and retrieved as an environment variable.
+**Type:** String
+**Default:** n/a
+**Note:** Store and retrieve your `apiKey` as an environment variable.
 
 ### `apiSecret` (required)
 
 The API Secret of your Cloudinary account. You'll find it in your [Cloudinary console](https://cloudinary.com/console/).
 
-**Type:** String  
-**Default:** n/a  
-**Note:** This should be stored and retrieved as an environment variable.
+**Type:** String
+**Default:** n/a
+**Note:** Store and retrieve your `apiSecret` as an environment variable.
 
 ### `resourceType`
 
 The _resource_ types to include when pulling data from Cloudinary.
 
-**Type:** String  
-**Default:** `image`  
-**Valid:** `image`, `raw` and `video`  
-**Note:** Use the video resource type for all video and audio files, such as `.mp3` and `.mp4`.
+**Type:** String
```

Comment:
You have removed the double space at the end of all of these, removing the line break.

Reply:
I added the double space and prettier or something deleted them all when I saved. Can you add them?